tests/test_detectionlists_actions.py

```diff
@@ -1,8 +1,6 @@
 from pytest import fixture
```

Comment:
That tests that end in `_sets_success_status` should probably be changed to `_sets_success_message()` since that seems to be more the focus of the tests now.